cmd/follow.go

```diff
@@ -14,31 +14,36 @@ type options struct {
 	cfg *settings.Config
 }
 
+//followProject gets the remote data and attempts to follow its git project
 func followProject(opts options) error {
 
 	remote, err := git.InferProjectFromGitRemotes()
-
 	if err != nil {
 		return errors.Wrap(err, errorMessage)
 	}
 
-	vcsShort := "gh"
-	if remote.VcsType == "BITBUCKET" {
-		vcsShort = "bb"
-	}
-	res, err := api.FollowProject(*opts.cfg, vcsShort, remote.Organization, remote.Project)
-	if err != nil {
-		return err
-	}
-	if res.Followed {
-		fmt.Println("Project successfully followed!")
-	} else if res.Message == "Project not found" {
-		fmt.Println("Unable to determine project slug for CircleCI (slug is case sensitive).")
-	}
+	//check that project url contains github or bitbucket; our legacy vcs
+	if remote.VcsType == git.GitHub || remote.VcsType == git.Bitbucket {
+		vcsShort := "gh"
+		if remote.VcsType == "BITBUCKET" {
+			vcsShort = "bb"
+		}
+		res, err := api.FollowProject(*opts.cfg, vcsShort, remote.Organization, remote.Project)
+		if err != nil {
+			return err
+		}
+		if res.Followed {
+			fmt.Println("Project successfully followed!")
+		} else if res.Message == "Project not found" {
+			fmt.Println("Unable to determine project slug for CircleCI (slug is case sensitive).")
+		}
 
-	return nil
+	}
```

Comment:
I _think_ you might need an `else` clause here otherwise the conditions on lines 36 and 38 would print messages and then print an error

```suggestion
		return nil
	} else {
		//if not warn user their vcs is not supported
		return errors.New(errorMessage)
	}
```

Reply:
Yup, added